scripts/rf_diagnostic_data.py

```diff
@@ -119,13 +124,13 @@
         path = os.path.join(args.directory, filename)
     with open(path, "wb") as file:
         file.write(data)
-    print("Saved diagnostic data to '{}'".format(path))
+    logger.info("Saved diagnostic data to '{}'".format(path))
 except Exception as e:
-    if args.debug:
-        logger.error("Caught exception:\n\n{}\n".format(traceback.format_exc()))
+    logger.debug("Caught exception:\n\n{}\n".format(traceback.format_exc()))
     exit_code = 1
-    print(e)
+    logger.info(e)
 finally:
     # Log out
-    redfish_utilities.logout(redfish_obj)
+    if not args.session_token:
```

Comment:
Could we add a common method for this while we're here? Just thinking about potential future "what if" scenarios where we might need to add more logic around logging out from the script.

Reply:
Sure, we don't have to solve it today since this is a pretty minor change